configs/nextflow.hopper.config

```diff
@@ -44,17 +43,22 @@ params {
   rCRS_fasta        = "${refpath}/fasta/split/M.fa"
 
   // VEP REFERENCES AND ANNOTATION DBS //
+  VEP_DIR           = "${refpath}/vep/.vep111.0"
```

Comment:
I think we should consider renaming the dir to `vep111.0`. It has confused me many times while looking through the annotation files where the vep files are